src/common/module.c

```diff
@@ -40,7 +40,8 @@ static int run_command(const char *command)
 
 	if (status < 0) {
 		char error_buf[80];
-		strerror_r(errno, error_buf, sizeof(error_buf));
+		char* dummy = strerror_r(errno, error_buf, sizeof(error_buf));
```

Comment:
could also put
```c++
(void) strerror_r(errno, error_buf, sizeof(error_buf));
```
to silence the warning.

wondering why the `strerror_r()` call didn't pop up.